.changeset/thick-pumpkins-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': major
+---
+
+`Initializable`: Allow versions greater than 256.
```

Comment:
```suggestion
`Initializable`: Use the namespaced storage pattern to avoid putting critical variables in slot 0. Allow reinitializer versions greater than 256.
```